tests/net/socket/socketpair/src/test_socketpair_unsupported_calls.c

```diff
@@ -58,6 +58,9 @@ void test_socketpair_unsupported_calls(void)
 			"listen should set errno to EINVAL");
 
 		res = accept(sv[i], (struct sockaddr *)&addr, &len);
```

Comment:
IMO we don't need to do this; it's test code, and if the accept() didn't fail it's fine to leak the descriptor.

Reply:
Ok. I wasn't sure if it was mandatory. Will have a look again soon

Reply:
dropped the commit